public/index.html

```diff
@@ -7,5 +7,8 @@
 <body>
     <div id='app'></div>
     <script src='/bundle.js'></script>
+
+    <!--for local version-->
+    <!--<script src='../dist/bundle.js'></script>-->
```

Comment:
You can use html-webpack-plugin that can attach produced bundles to the HTML depending on configuration https://github.com/jantimon/html-webpack-plugin

Reply:
I hope I got you right.

Reply:
ok. then I need to write template for div#app which is pretty much the same. 
so I will leave it as is.